src/checkout/context/checkout.tsx

```diff
@@ -119,33 +119,47 @@ export const CheckoutProvider: FC<Props> = React.memo(({children}) => {
   })
 
   const getZuoraParams = useCallback(async () => {
-    const response = await getCheckoutZuoraParams()
-    if (response.status !== 200) {
-      throw new Error(getErrorMessage(response))
-    }
+    try {
+      const response = await getPaymentForm({form: 'checkout'})
+      if (response.status !== 200) {
+        throw new Error(getErrorMessage(response))
+      }
 
-    setZuoraParams(response.data as CreditCardParams)
+      // TODO(ariel): remove type casting here when Billing is refactored and integrated
+      setZuoraParams(response.data as CreditCardParams)
+    } catch (error) {
+      // TODO(ariel): should we notify or report the error?
```

Comment:
open question about how we should handle failures to the Zuora form? Right now a white iframe is displayed with the error message,  is that enough? Quartz does the same but can we do better?